source/getting-started.txt

```diff
@@ -0,0 +1,72 @@
+.. _relational-migrator-getting-started:
+
+===============
+Getting Started
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Relational Migrator allows you to bring your relational workloads 
+to MongoDB. 
+
+To get familiar with Relational Migrator capabilities, 
+features, and use cases review the follow resources:
+
+- :ref:`<supported-databases>`
+-  :ref:`<rm-migration-scenarios>`
+- `Relation Migrator Learning Byte <https://learn.mongodb.com/courses/mongodb-relational-migrator>`__
+- `Relational Migrator NYC.Local Product Demo <https://www.youtube.com/watch?v=-ND6_m397OM&ab_channel=MongoDB>`__
+
+Steps
+-----
+
+The following steps are a high level workflow of how you can use 
+Relational Migrator to get your data into MongoDB. These steps do not 
+cover all features or capabilities of the tool.
```

Comment:
(info typing)
Consider removing this intro para. You can move anything critical to the short description!